custom_components/tahoma/tahoma_device.py

```diff
@@ -119,7 +119,7 @@ def device_state_attributes(self):
     def device_info(self):
         """Return device registry information for this entity."""
         return {
-            "identifiers": {(DOMAIN, self.unique_id)},
+            "identifiers": {(DOMAIN, self.unique_id.split("#", 1)[0])},
             "manufacturer": "Somfy",
             "name": self.name,
             "model": self.tahoma_device.widget,
```

Comment:
name and model should not stay I guess, at least in the Thermostat, they're not the same for all 3 entities.